client/styles/components/_console.scss

```diff
@@ -86,22 +86,26 @@
 
 .preview-console__clear {
 	@include themify() {
-		@extend %link;
-		color: getThemifyVariable('secondary-text-color');
-		&:hover {
-			color: getThemifyVariable('heavy-text-color');
+	  @extend %link;
+	  color: getThemifyVariable('secondary-text-color');
+	  &:hover {
+		color: $p5-light-pink; 
+  
+		@if (getThemifyVariable('logo-color') == $yellow) {
+		  color: $yellow; 
 		}
+	  }
 	}
 	background: transparent;
 	border: none;
 	padding-right: #{math.div(10, $base-font-size)}rem;
 	.preview-console--collapsed & {
-		display: none;
+	  display: none;
```

Comment:
Please remove the extra spaces on lines `103` and `105`, and the extra change on line `111`, since they introduce new changes that aren't related to this PR!